### DISCUSSION TITLE
[WASM] Unwanted and wrong framework dependency Microsoft.AspNetCore.App in project

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#13130] [Q&A] [answered]
My WASM project in my solution is holding a dependency on Microsoft.AspNetCore.App which 
- is wrong
- is showing an error

I'm seeing the same wrong/erroneous framework dependency in a blank, new UNO project created as per [here](https://platform.uno/docs/articles/get-started-dotnet-new.html?tabs=net7%2Cwindows):
`dotnet new unoapp -preset=blank -o test`

How could I remove that wrong/erroneous dependency?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. This is a known issue (https://github.com/unoplatform/Uno.Wasm.Bootstrap/issues/460) that we cannot fix at this time.

**Reply by DierkDroth** [reply]
Thanks for your feedback @jeromelaban
